README.md

```diff
@@ -35,6 +35,10 @@ defraUtils.updateGithubCommitStatus('Build started', 'PENDING')
 ```
 ## Functions
 
+### getCSProjVersion
+
+Returns the project version from the `[projectName].csproj` file
```

Comment:
I'd give more detail about how to use this as unlike the Node.js equivalent a parameter is needed.  We should explain what it is.